charts/api/templates/_helpers.tpl

```diff
@@ -64,6 +64,10 @@ Common lists of environment variables
 - name: WBSTACK_SUMMARY_INACTIVE_THRESHOLD
   value: {{ .Values.wbstack.summaryInactiveThreshold | quote }}
 {{- end }}
+{{- if .Values.wbstack.summaryCreationRateRanges }}
+- name: WBSTACK_SUMMARY_CREATION_RATE_RANGES
+  value: {{ join ',' .Values.wbstack.summaryCreationRateRanges | quote }}
```

Comment:
```suggestion
  value: {{ join "," .Values.wbstack.summaryCreationRateRanges | quote }}
```

apparently using double quotes here fixes the error I mentioned https://github.com/wbstack/charts/pull/119/files#r1251824807